app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   before_action :set_sentry_user
+  before_action :check_teacher_admin
```

Comment:
What is the point of this function rather than to write two separate before_action for is teacher and is admin?

Reply:
cuz nav_bar looks at @is_teacher and @is_admin. Need to call them to set them up